py-polars/tests/unit/operations/map/test_inefficient_map_warning.py

```diff
@@ -211,6 +231,7 @@ def test_parse_apply_functions(col: str, func: str, expr_repr: str) -> None:
                 "b": ["AB", "cd", "eF"],
                 "c": ['{"a": 1}', '{"b": 2}', '{"c": 3}'],
                 "d": ["2020-01-01", "2020-01-02", "2020-01-03"],
+                "e": [1.5, 2.4, 3.1],
```

Comment:
adding an extra float column to the test df as some of these functions (e.g. floor) are only supported for floats